docs/index.md

```diff
@@ -21,7 +21,9 @@ Some links to pages that may be useful are:
 
 ```{eval-rst}
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 4
+   :caption: Contents
```

Comment:
```suggestion
   :maxdepth: 2
```

Max depth of 2 is a bit less like information overload. Caption of "Contents" is redundant and doesn't need to be specified, since there is already a markdown header that says "Contents"/"Table of Contents".